tests/FSharp.Compiler.UnitTests/FSharp.Compiler.UnitTests.fsproj

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netcoreapp1.0</TargetFrameworks>
+    <PackageTargetFallback>portable-net45+win8+wp8+wpa81</PackageTargetFallback>
+    <OutputType>Library</OutputType>
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <Optimize>false</Optimize>
+    <Tailcalls>false</Tailcalls>
+    <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp1.0'">$(DefineConstants);coreclr</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)' == 'net46'">$(DefineConstants);net40</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="NunitHelpers.fs" />
+    <Compile Include="CompilerTestHelpers.fs" />
+    <Compile Include="ManglingNameOfProvidedTypes.fs" />
+    <Compile Include="HashIfExpression.fs" />
+    <Compile Include="ProductVersion.fs" />
+    <Compile Include="EditDistance.fs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NUnitLite" Version="$(NUnitLitePackageVersion)" />
```

Comment:
to convert nunit tests, is maybe better to use the nunit and .net test sdk, not nunit lite (is not the way forward for .net core).

info in https://github.com/nunit/dotnet-new-nunit

just install the nunit template, and `dotnet new nunit` to see an example proj.

like that, is just vstest (so will work in VS , `dotnet test` and others).